src/packages/frontend/site-licenses/admin/license.tsx

```diff
@@ -142,6 +142,12 @@ export const License: React.FC<Props> = (props: Props) => {
     actions.set_edit(license.get("id"), field, new_val);
   }
 
+  function on_date_change(field, _date, dateString): void {
+    // For date fields, use the dateString instead of the Day.js object
+    // to ensure proper serialization for PostgreSQL
+    actions.set_edit(license.get("id"), field, dateString || null);
```

Comment:
Why use `dateString || null` instead of `dateString ?? null` which seems more idiomatic in modern times.  Who uses || anymore?

Reply:
So, there is an issue, where unsetting the expire date field does not set it to null in the database. The idea is to send a "null" value instead of an empty string. Doesn't work, though.